### PR TITLE
3.0.0-beta0008

### PR DESCRIPTION
**Pull request by krankenbro**
Updated Dependencies;
Fixed issue with Gitversion and Nuke.Common;

### CODE REVIEW

File: build/Build.cs [hunk 1]
```diff
@@ -302,8 +307,8 @@ void ErrorLogger(OutputType type, string text)
           {
               //dotnet %userprofile%\.nuget\packages\swashbuckle.aspnetcore.cli\4.0.1\lib\netcoreapp2.0\dotnet-swagger.dll tofile --output swagger.json bin/Debug/netcoreapp2.2/VirtoCommerce.Platform.Web.dll VirtoCommerce.Platform
```
Comment (akak1977) [inline build/Build.cs]: I think it's need to remove these redundant comments

File: build/_build.csproj [hunk 2]
```diff
@@ -15,16 +15,17 @@
...
-        <PackageReference Include="GitVersion.CommandLine" Version="5.1.3">
+    <ItemGroup>
+        <PackageReference Include="GitVersion.CommandLine" Version="5.1.2">
           <PrivateAssets>all</PrivateAssets>
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-        </PackageReference>      
+        </PackageReference>
         <PackageReference Include="Nuke.Common" Version="0.23.6" />
-        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.0.0-beta0004" />
+        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.0.0-rc3-0141" />
```
Comment (akak1977) [inline build/_build.csproj]: Better to make there project reference to prevent copying obsolete assemblies to publish

File: build/Build.cs [hunk 3]
```diff
@@ -300,10 +304,8 @@ void ErrorLogger(OutputType type, string text)
           .Requires(() => !IsModule)
           .Executes(() =>
           {
-              //dotnet %userprofile%\.nuget\packages\swashbuckle.aspnetcore.cli\4.0.1\lib\netcoreapp2.0\dotnet-swagger.dll tofile --output swagger.json bin/Debug/netcoreapp2.2/VirtoCommerce.Platform.Web.dll VirtoCommerce.Platform
-              //better use in the future a .Net Global Tool https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/master/README-v5.md#retrieve-swagger-directly-from-a-startup-assembly
-              var swashbucklePackage = NuGetPackageResolver.GetGlobalInstalledPackage("swashbuckle.aspnetcore.cli", "4.0.1", "dotnet-swagger.dll");
-              var swashbucklePath = swashbucklePackage.Directory / "lib" / "netcoreapp2.0" / "dotnet-swagger.dll";
+              var swashbucklePackage = NuGetPackageResolver.GetGlobalInstalledPackage("swashbuckle.aspnetcore.cli", "5.0.0", "dotnet-swagger.dll");
+              var swashbucklePath = swashbucklePackage.Directory / "tools" / "netcoreapp3.0" / "any" / "dotnet-swagger.dll";
```
Comment (akak1977) [inline build/Build.cs]: It's not good to hard-code paths like "netcoreapp3.0".  It should be better to search for dll.

Comment (yecli) [reply build/Build.cs]: @krankenbro Will this comment be addressed?

Comment (krankenbro) [reply build/Build.cs]: @yecli fixed

**Pull request merged**